backend/mora/service/employee.py

```diff
@@ -226,6 +231,9 @@ def create_employee_relation(employee_uuid):
     :<jsonarr string engagement_type: The engagement type
     :<jsonarr object validity: The validities of the created object.
 
+    The parameters ``job_function`` and ``engagement_type`` should contain
+    UUIDs obtained from their respective facet endpoints
```

Comment:
For this, and the other similar references, we should probably some of this text a hyperlink to `/o/<uuid:orgid>/f/`. Also, perhaps we should consider an example instead of merely an unspecific _“their respective”_?